db/seeds.rb

```diff
@@ -102,15 +104,29 @@ class SeedAdminENVError < StandardError
     applications = []
 
     2.times do
-      r.rand(1..NUM_USERS).times do
+      r.rand(1..NUM_USERS).times do |i|
 
         next unless (company_number = get_company_number(r))
 
-        ma = MembershipApplication.new(first_name: FFaker::NameSE.first_name,
-                                       last_name: FFaker::NameSE.last_name,
-                                       contact_email: FFaker::InternetSE.free_email,
+        u = users[r.rand(0..NUM_USERS-1)]
+
+        #  If the user already has a membership application, use the same names.
+          # (They would only use different name if they made a mistake and submitted a whole new application.  We won't worry about that case here.)
+        if (m = MembershipApplication.find_by(user_id: u.id))
+          first_n = m.first_name
+          last_n = m.last_name
+        else
+          first_n = FFaker::NameSE.first_name
+          last_n = FFaker::NameSE.last_name
+        end
+
+        # Every 6th user has a different membership application contact email than the email they use to log in to SHF
+        ma = MembershipApplication.new(first_name: first_n,
+                                       last_name: last_n,
+                                       contact_email: (i.divmod(6).last == 0 ? FFaker::InternetSE.free_email : u.email),
```

Comment:
This is fine as-is, even though `divmod` does more work than we need.  A simpler approach would be: 
```
i % 6 == 0 ? ....
```